README.md

```diff
@@ -97,14 +97,37 @@ $ balena run -it -v ${appid}_resin-data:/mnt alpine \
 
 ## Networking
 ### Shared physical device
-First, create a bridge, and assign the physical interface as a slave. This can be done on the host OS using nmcli:
+
+Check your network before you wreck your network:
+
+Find the ethernet name for example something like this eno1 or enp89s0 
+`` 
+$ ip a
+``
+
+First, create a bridge on thee ethernet port preferably(wifi will require lan and dhcp server), and assign the physical interface as a slave. This can be done on the host OS using nmcli:
 
 ```
 $ nmcli connection add type bridge ifname qemu0
 $ nmcli connection add type bridge-slave ifname eno1 master qemu0
 ```
 
+If you screw it up, then delete 
```

Comment:
The tone is inconsistent with the rest of the readme, though this is good information to have, thanks for including it.